src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -2363,8 +2478,7 @@ export default class AircraftModel {
                     headingDisplay,
                     destinationDisplay,
                     isFollowingSid,
-                    isFollowingStar,
-                    fixRestrictions
+                    isFollowingStar
```

Comment:
be sure to update this branch with develop, all of these aircraft strip methods have been removed